testing/testcafe/tests/scheduler/dragAndDrop/cancelAppointmentDrag.ts

```diff
@@ -0,0 +1,58 @@
+import { ClientFunction } from 'testcafe';
+import { safeSizeTest } from '../../../helpers/safeSizeTest';
+import createScheduler from './init/widget.setup';
+import url from '../../../helpers/getPageUrl';
+import Scheduler from '../../../model/scheduler';
+
+fixture`Cancel appointment D-n-D`
```

Comment:
Minor comment.

Let's rename this fixture:
```suggestion
fixture`Cancel appointment Drag-and-Drop`
```